hack/make-rules/test-cmd-util.sh

```diff
@@ -1050,48 +1050,6 @@ run_kubectl_create_filter_tests() {
   set +o errexit
 }
 
-run_kubectl_apply_deployments_tests() {
```

Comment:
@kow3ns Should we remove these tests? They verify kubectl apply propagates to user defined null values, which i think is no more a valid use case because template label changes do not propagate to spec selector anymore after removing selector and label defaulting codes.